### DISCUSSION TITLE
My webpage has lost its custom font

### DISCUSSION THREAD

**Discussion by Mngadinm** [community/community#54132] [Programming Help]
### Select Topic Area

Question

### Body

Hi Guys,

could you please assist here, im still learning. i uploaded my page but the font that i got from google fonts is missing on my github page. Please check it out and advise what can i do to fix this:

my rep:
https://github.com/Mngadinm/valleyacademy.github.io
my page:
https://mngadinm.github.io/valleyacademy.github.io/

many thanks

**Comment by hrishikesh-kocharekar-dev** [comment]
it appears that the issue is with the path you have specified for your custom font in your CSS file. Currently, your CSS file specifies the following path for the font: 
url('fonts/UnifrakturMaguntia-Regular.ttf') format('truetype');

However, the font file is actually located in the following path in your repository:
/assets/fonts/UnifrakturMaguntia-Regular.ttf

To fix the issue, you can update the path in your CSS file to match the correct location of the font file:
url('../assets/fonts/UnifrakturMaguntia-Regular.ttf') format('truetype');

This should ensure that the custom font is loaded correctly on your webpage.

After updating the CSS file, commit and push the changes to your GitHub repository, and the changes should be reflected on your webpage.
